test/extensions/filters/http/ext_authz/ext_authz_integration_test.cc

```diff
@@ -871,9 +901,61 @@ TEST_P(ExtAuthzGrpcIntegrationTest, DownstreamHeadersOnSuccess) {
   cleanup();
 }
 
+TEST_P(ExtAuthzGrpcIntegrationTest, TimeoutFailClosed) {
+  GrpcInitializeConfigOpts opts;
+  opts.failure_mode_allow = false;
+  opts.timeout_seconds = 1;
```

Comment:
For the sake of making integration tests not take any longer than they already do, can you decrease the timeout to something like 10ms?

Reply:
Done.